src/dissocPath.js

```diff
@@ -0,0 +1,37 @@
+var _curry2 = require('./internal/_curry2');
+var _slice = require('./internal/_slice');
+var assoc = require('./assoc');
+var dissoc = require('./dissoc');
+var is = require('./is');
+var split = require('./split');
+
+
+/**
+ * Makes a shallow clone of an object, omitting the property at the
+ * given path. Note that this copies and flattens prototype properties
+ * onto the new object as well.  All non-primitive properties are copied
+ * by reference.
+ *
+ * @func
+ * @memberOf R
+ * @category Object
+ * @sig String -> {k: v} -> {k: v}
+ * @param {String} path the dot-delimited path to set
+ * @param {Object} obj the object to clone
+ * @return {Object} a new object without the property at path
+ * @example
+ *
+ *      var obj1 = {a: {b: 1, c: 2, d: {e: 3}}, f: {g: {h: 4, i: 5, j: {k: 6, l: 7}}}, m: 8};
+ *      var obj2 = R.dissocPath('f.g.i', obj1);
+ *      //=> {a: {b: 1, c: 2, d: {e: 3}}, f: {g: {h: 4, j: {k: 6, l: 7}}}, m: 8}
```

Comment:
Could you simplify this example? It's a bit hard to see how the input and output differ.